test/LighthouseStringifyExtension.test.ts

```diff
@@ -16,9 +16,9 @@
 
 import snapshot from 'snap-shot-it';
 
-import { stringify } from '../../src/stringify.js';
-import { LighthouseStringifyExtension } from '../../src/lighthouse/LighthouseStringifyExtension.js';
-import { AssertedEventType, StepType, UserFlow } from '../../src/Schema.js';
```

Comment:
nit: I prefer having this test file in its own folder because it mirrors the directory structure of the source code.